inference-engine/tests/functional/plugin/shared/include/single_layer_tests/reshape.hpp

```diff
@@ -8,8 +8,8 @@
 
 namespace LayerTestsDefinitions {
 
-TEST_P(ReshapeLayerTest, CompareWithRefsDynamicBath) {
+TEST_P(ReshapeLayerTestRevise, CompareWithRefsDynamicBath) {
```

Comment:
@pelszkow I think we should bring the original name `ReshapeLayerTest` here if we can. How did you resolve this dependency in plugins ? Did you removed this dependency entirely or now plugins depend on 'ReshapeLayerTestRevise' ?

Reply:
In plugins I've switched to new name of SLT ([ARM PR-155](https://github.com/openvinotoolkit/openvino_contrib/pull/155)).

If we want to have previous name we should (in current approach):
- copy `ReshapeLayerTestRevise` with new name `ReshapeLayerTest` in OV 
- once again update plugins
- drop `ReshapeLayerTestRevise`

There is also a option to drop dependency in plugins before start to work on SLT in OV or stay with "Revise" name as temporary solution and create task to fix it in the future.

@ilyachur , what do you preferer, stay with "Revise" SLT name or fix it by plugins team by dropping dependency to OV SLT?

Reply:
@pelszkow @jdanieck Honestly from my point of view will be better to have only `ReshapeLayerTest` because it is aligned with other test names, `Revise` word can cause some questions (like, for what operations should we use revised tests?) the original name is more clear for me.